samples/subsys/usb/uvc/boards/stm32n6570_dk.conf

```diff
@@ -0,0 +1,3 @@
+# Enough for 2592x1944 YUYV frames
+CONFIG_VIDEO_BUFFER_POOL_SZ_MAX=10100000
+CONFIG_MAIN_STACK_SIZE=2048
```

Comment:
These are three identical configuration overlays. And I fear that would be just the beginning of the madness. I would accept more sophisticated configuration overlay for one or other board, as an example, but I am absolutely against trivial overlay files. You have to find another way to configure your boards. You can pass these configuration to the build command and add documentation to the sample how to do it.